gson/src/main/java/com/google/gson/internal/LinkedHashTreeMap.java

```diff
@@ -762,6 +762,9 @@ void add(Node<K, V> node) {
     Node<K, V> lastReturned = null;
     int expectedModCount = modCount;
 
+    LinkedTreeMapIterator() {
```

Comment:
... because this is a private class. Alternative would be to remove `private` on the `LinkedTreeMapIterator` class declaration.